src/test/java/uk/gov/companieshouse/mapper/CompanyOfficerMapperTest.java

```diff
@@ -0,0 +1,148 @@
+        final CompanyOfficerApi companyOfficerApiThree = new CompanyOfficerApi();
+        companyOfficerApiThree.setName(DIRECTOR_NAME_THREE);
+        companyOfficerApiThree.setOfficerRole(OfficerRoleApi.DIRECTOR);
+        companyOfficerApiThree.setAppointedOn(LocalDateTime.now().minusWeeks(1).toLocalDate());
+        final List<CompanyOfficerApi> companyOfficers = Arrays.asList(
+                companyOfficerApiOne,
+                companyOfficerApiTwo,
+                companyOfficerApiThree
+        );
+
+        final boolean hasOverHalfDirectorsSelected =
+                mapper.areMajorityOfCompanyOfficersSelected(companyOfficers, selectedDirectors);
+
+        assertTrue(hasOverHalfDirectorsSelected);
+    }
+
+    @Test
+    public void mapCompanyOfficersToIsOverHalfSelected_allDirectorsSelected_returnsTrue() {
+        final DissolutionDirector directorOne = new DissolutionDirector();
+        directorOne.setName(DIRECTOR_NAME);
+        directorOne.setEmail(DIRECTOR_EMAIL);
+        final DissolutionDirector directorTwo = new DissolutionDirector();
+        directorTwo.setName(DIRECTOR_NAME_TWO);
+        directorTwo.setEmail(DIRECTOR_EMAIL_TWO);
+        final List<DissolutionDirector> selectedDirectors = Arrays.asList(directorOne, directorTwo);
+
+        final CompanyOfficerApi companyOfficerApiOne = new CompanyOfficerApi();
+        companyOfficerApiOne.setName(DIRECTOR_NAME);
+        companyOfficerApiOne.setOfficerRole(OfficerRoleApi.DIRECTOR);
+        companyOfficerApiOne.setAppointedOn(LocalDateTime.now().minusWeeks(1).toLocalDate());
+        final CompanyOfficerApi companyOfficerApiTwo = new CompanyOfficerApi();
+        companyOfficerApiTwo.setName(DIRECTOR_NAME_TWO);
+        companyOfficerApiTwo.setOfficerRole(OfficerRoleApi.DIRECTOR);
+        companyOfficerApiTwo.setAppointedOn(LocalDateTime.now().minusWeeks(1).toLocalDate());
+        final List<CompanyOfficerApi> companyOfficers = Arrays.asList(
+                companyOfficerApiOne,
+                companyOfficerApiTwo
+        );
+
+        final boolean hasOverHalfDirectorsSelected =
+                mapper.areMajorityOfCompanyOfficersSelected(companyOfficers, selectedDirectors);
+
+        assertTrue(hasOverHalfDirectorsSelected);
+    }
+
+    @Test
+    public void mapCompanyOfficersToIsOverHalfSelected_noDirectorsSelected_returnsFalse() {
+        final List<DissolutionDirector> selectedDirectors = Collections.emptyList();
+        final List<CompanyOfficerApi> companyOfficers = DissolutionFixtures.generateCompanyOfficerList();
+
+        final boolean hasOverHalfDirectorsSelected =
+                mapper.areMajorityOfCompanyOfficersSelected(companyOfficers, selectedDirectors);
+
+        assertFalse(hasOverHalfDirectorsSelected);
+    }
+
+    @Test
+    public void mapCompanyOfficersToIsOverHalfSelected_wrongDirectorsSelected_returnsFalse() {
+        final List<CompanyOfficerApi> companyOfficers = DissolutionFixtures.generateCompanyOfficerList();
```

Comment:
looking at this test alone I have no idea what is in `companyOfficers` and how this impacts the behaviour of the test. Again, I don't think we should every rely on the content of fixtures for our assertions.